policyengine_api/routes/simulation_analysis_routes.py

```diff
@@ -41,41 +41,28 @@ def execute_simulation_analysis(country_id):
     )
     audience = payload.get("audience", "")
 
-    try:
-        analysis = simulation_analysis_service.execute_analysis(
-            country_id,
-            currency,
-            selected_version,
-            time_period,
-            impact,
-            policy_label,
-            policy,
-            region,
-            relevant_parameters,
-            relevant_parameter_baseline_values,
-            audience,
-        )
+    analysis = simulation_analysis_service.execute_analysis(
+        country_id,
+        currency,
+        selected_version,
+        time_period,
+        impact,
+        policy_label,
+        policy,
+        region,
+        relevant_parameters,
+        relevant_parameter_baseline_values,
+        audience,
+    )
 
-        # Create streaming response
-        response = Response(
-            stream_with_context(analysis),
-            status=200,
-        )
+    # Create streaming response
+    response = Response(
+        stream_with_context(analysis),
+        status=200,
+    )
 
-        # Set header to prevent buffering on Google App Engine deployment
-        # (see https://cloud.google.com/appengine/docs/flexible/how-requests-are-handled?tab=python#x-accel-buffering)
-        response.headers["X-Accel-Buffering"] = "no"
+    # Set header to prevent buffering on Google App Engine deployment
+    # (see https://cloud.google.com/appengine/docs/flexible/how-requests-are-handled?tab=python#x-accel-buffering)
+    response.headers["X-Accel-Buffering"] = "no"
```

Comment:
**suggestion, non-blocking** again strongly suggest we don't iterate again over this, but I think this would be cleaner as a decorator [like this](https://flask-headers.readthedocs.io/en/latest/). I don't think that is necessarily the right library so that would require a little research.

Suggest we look into how to set headers as part of integrating parsing/serialization automation.